src/Ngsa.DataService/Controllers/ResultHandler.cs

```diff
@@ -49,22 +45,19 @@ public static async Task<IActionResult> Handle<T>(Task<T> task, NgsaLog logger)
                 // log and return Cosmos status code
                 if (ce.StatusCode == HttpStatusCode.NotFound)
                 {
-                    logger.EventId = new EventId((int)ce.StatusCode, string.Empty);
-                    logger.LogWarning(logger.NotFoundError);
+                    logger.LogWarning(new EventId((int)ce.StatusCode, string.Empty), nameof(Handle), logger.NotFoundError);
+
                     return CreateResult(logger.NotFoundError, ce.StatusCode);
                 }
 
-                logger.EventId = new EventId((int)ce.StatusCode, "CosmosException");
-                logger.Data.Add("CosmosActivityId", ce.ActivityId);
-                logger.LogError($"CosmosException: {ce.Message}", ce);
+                logger.LogError(new EventId((int)ce.StatusCode, "CosmosException"), nameof(Handle), $"CosmosActivityId: {ce.ActivityId}", ex: ce);
 
                 return CreateResult(logger.ErrorMessage, ce.StatusCode);
             }
             catch (Exception ex)
             {
                 // log and return exception
-                logger.EventId = new EventId((int)HttpStatusCode.InternalServerError, "Exception");
-                logger.LogError($"Exception: {ex.Message}", ex);
+                logger.LogError(new EventId((int)HttpStatusCode.InternalServerError, "Exception"), nameof(Handle), "Exception: {ex.Message}", ex: ex);
```

Comment:
Do the nameof(Handle) print something like "System.Activities.Handle" as part of the error log?  If so, we use the name of whatever T is for the log?